tests/PySys/call_tedge_connect_c8y_--test_positive/run.py

```diff
@@ -0,0 +1,43 @@
+import sys
+
+sys.path.append("environments")
+from environment_c8y import EnvironmentC8y
+
+"""
+Test `tedge connect c8y --test` (successful case):
+
+Given a configured system with configured certificate
+When we setup EnvironmentC8y
+When we execute `sudo tedge connect c8y --test`
+When we validate stdout
+When we cleanup EnvironmentC8y
+Then we find a successful message in stdout
+Then the test has passed
+
+"""
+
+
+class RestartBridge(EnvironmentC8y):
```

Comment:
This class name doesn't really reflect what the test is doing.

Reply:
Will change it to `TedgeConnectTestPositive`.

Reply:
Hi @albinsuresh I would recommend to continue with snake case names as the other directory names are also snake case